.travis.yml

```diff
@@ -41,7 +34,7 @@ jobs:
         - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
         - if [[ ! $(php -m | grep -si xdebug) ]]; then echo "xdebug required for coverage"; exit 1; fi
       script:
-        - ./vendor/bin/phpunit --coverage-clover clover.xml
+        - bin/phpunit --coverage-clover clover.xml
```

Comment:
as mentioned in https://github.com/webonyx/graphql-php/pull/333/files#r213260516, this should be reverted